dags/python_scripts/atd_vzd_update_cr3_locations.py

```diff
@@ -71,7 +71,10 @@ def make_update() -> dict:
         }
     )
     response.encoding = "utf-8"
-    return response.json()
+    if response.status_code == 200:
```

Comment:
i take it you're getting a `504` on timeout? i asked b/c you can set a request [`timeout`](https://requests.readthedocs.io/en/latest/user/quickstart/#timeouts) but that won't help if the serve is bombing.

is if the request (not the server) times out, then `requests` will raise a timeout exception and there will be no `response` object inspect. based on the Airflow errors that does not appear to be a concern but just wanted to mention it.

Reply:
So far it has been a 504 timeout everytime i get an error running locally. I guess if other errors appear I can expand this to handle other errors? or should I just get in front of it now....

Reply:
👍 i think this is fine—this whole thing is wrapped in a try/except.